src/components/PastWinners/index.tsx

```diff
@@ -0,0 +1,79 @@
+                    <FontAwesomeIcon icon={faLaptopCode} className="fa winner-icon" />
+                    <h5 className="winner-title">CyberSecurity</h5>
+                    <h6 className="winner-group">Repul.ink</h6>
+                </a>
+                <a href="https://devpost.com/software/sustainably-yours" target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon={faPalette} className="fa winner-icon" />
+                    <h5 className="winner-title">Best in Design</h5>
+                    <h6 className="winner-group">Sustainably Yours</h6>
+                </a>
+                <a href="https://devpost.com/software/uc-merced-basketball-analysis" target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon={faSchool} className="fa winner-icon" />
+                    <h5 className="winner-title">Best UC Merced-Related Hack</h5>
+                    <h6 className="winner-group">UC Merced Basketball Analysis</h6>
+                </a>
+                <a href="https://devpost.com/software/esose" target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon={faSearch} className="fa winner-icon" />
+                    <h5 className="winner-title">Tech for Social Good</h5>
+                    <h6 className="winner-group">Vacchecker</h6>
+                </a>
+                <a href="https://devpost.com/software/3d-for-education" target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon={faMicrochip} className="fa winner-icon" />
+                    <h5 className="winner-title">Best Hardware Hack</h5>
+                    <h6 className="winner-group">EcoSafe</h6>
+                </a>
+                <a href="https://devpost.com/software/blood-buds" target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon={faMeteor} className="fa winner-icon" />
+                    <h5 className="winner-title">Moonshot</h5>
+                    <h6 className="winner-group">Blood Buds</h6>
+                </a>
+                <a href="https://devpost.com/software/streetspots" target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon={faDigitalOcean} className="fa winner-icon" />
+                    <h5 className="winner-title">Best Domain Using Domain.com</h5>
+                    <h6 className="winner-group">StreetSpots.Online</h6>
+                </a>
+                <a href="https://devpost.com/software/idvision" target="_blank" rel="noopener noreferrer">
+                    <FontAwesomeIcon icon={faCloud} className="fa winner-icon" />
+                    <h5 className="winner-title">Best Use of Google Cloud</h5>
+                    <h6 className="winner-group">IdVision</h6>
+                </a>
+            </nav>
+        </section>
+    );
+};
+
+export default PastWinners;
```

Comment:
Past Winners was migrated to the `src/pages/Home`, please make your changes there.